From 9d41ecee6232613e1633850235921640dfa662c3 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Tue, 22 Sep 2020 14:53:44 +0800
Subject: [PATCH] Test case4 for Change guid value in Protocols

Description: Change guid value in Protocols section (to gEfiSimpleFileSystemProtocolGuid = { 0x9F3E5B22, 0x6459, 0x11D2, { 0x8E, 0x49, 0x00, 0xA0, 0xC9, 0xd8, 0x72, 0x3B }}) in MdePkg/MdePkg.dec which used in MdeModulePkg/Core/Dxe/DxeMain.inf.

Repo: edk2

Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 MdePkg/MdePkg.dec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MdePkg/MdePkg.dec b/MdePkg/MdePkg.dec
index 682e61cb88..a7a1219407 100644
--- a/MdePkg/MdePkg.dec
+++ b/MdePkg/MdePkg.dec
@@ -1431,7 +1431,7 @@
   gEfiLoadFile2ProtocolGuid       = { 0x4006c0c1, 0xfcb3, 0x403e, {0x99, 0x6d, 0x4a, 0x6c, 0x87, 0x24, 0xe0, 0x6d }}
 
   ## Include/Protocol/SimpleFileSystem.h
-  gEfiSimpleFileSystemProtocolGuid = { 0x964E5B22, 0x6459, 0x11D2, { 0x8E, 0x39, 0x00, 0xA0, 0xC9, 0x69, 0x72, 0x3B }}
+  gEfiSimpleFileSystemProtocolGuid = { 0x9F3E5B22, 0x6459, 0x11D2, { 0x8E, 0x49, 0x00, 0xA0, 0xC9, 0xd8, 0x72, 0x3B }}
 
   ## Include/Protocol/TapeIo.h
   gEfiTapeIoProtocolGuid         = { 0x1E93E633, 0xD65A, 0x459E, { 0xAB, 0x84, 0x93, 0xD9, 0xEC, 0x26, 0x6D, 0x18 }}
-- 
2.27.0.windows.1

